README.md

```diff
@@ -0,0 +1,10 @@
+# Gigantic Parser Module
+
+A Module to Parse Data from *Motiga's* **Gigantic**
+
+
+## Installation:
+
+```
+pip install -e .[development]
```

Comment:
Isnt the first part (``` pip install .```) the commands used to install, and the command to develop is ```python setup.py develop```? Just want to make sure I get that sorted out.